docs/architecture/decisions/0008-allow-non-signon-users-to-access-fact-check-view-via-a-jwt.md

```diff
@@ -0,0 +1,69 @@
+
+# 8. Allow non-signon users to access fact check view via a JWT
+
+Date: 2025-11-27
+
+Status: Accepted
+
+## Context
+
+In the [previous ADR], we proposed adding a fact check view as a Rails engine to allow users who do not have access to 
+signon to preview content securely.
+
+We needed to determine how to authenticate these external users without requiring signon credentials.
+
+## Decision
+
+We will use JSON Web Tokens (JWTs) for authentication, following the existing pattern used for draft content access in 
+the draft stack.
+
+### How it works
+
+1. **Token generation**: Each Edition will have a method to generate a JWT containing:
+    - The content ID
+    - A new UUID field on the edition (identifying the specific edition)
```

Comment:
this would this be a new autogenerated `editions.uuid` field? And we'd use a UUID here rather than the existing `id` to prevent enumeration? If that's correct, should we include this info?